frontend/pages/software/ManageSoftwarePage/_styles.scss

```diff
@@ -49,7 +49,7 @@
     }
 
     p {
-      color: $core-dark-blue-grey;
+      color: $ui-fleet-black-75;
```

Comment:
We need components for our headers, so we don't have separate styling on each page. Not in this PR, of course.

Reply:
agreed. im pretty sure we will be able to do this properly when mikeT adds the typography info to the styleguide.

Reply:
Yes, this ticket is only related to the fixes outlined in #8113 and #8386 -- this is a part of the listed fix "Remove old color that is no longer used"